conf/docker.conf

```diff
@@ -114,13 +114,25 @@
 
 #### END PARSE Section #####
 
-# Add event_time as kinesis doesn't support @time (Annotations logs only)
+# Add event_time as kinesis doesn't support @time (Annotations logs only) and time so we have consistent time field in all logs
 <filter docker.annotations>
     @type       record_transformer
     enable_ruby true
     <record>
       event_time        ${ record["@time"] != nil ? record["@time"] : "" }
+      time              ${ record["@time"] != nil ? record["@time"] : "" }
```

Comment:
Can we just make time ${event_time} since it is holding the same value as this filed?

Reply:
Unfortunately didn't work as part of the same filter.